llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -14,11 +14,12 @@
 //===----------------------------------------------------------------------===//
 
 #include "SYCLDeviceLibReqMask.h"
-#include "SYCLDeviceRequirements.h"
 #include "SYCLKernelParamOptInfo.h"
 #include "SpecConstants.h"
 #include "Support.h"
+#include "llvm/SYCLLowerIR/SYCLDeviceRequirements.h"
```

Comment:
This should be moved to the block below with other "llvm/SYCLLowerIR/*" includes.

Reply:
[Move include](https://github.com/intel/llvm/pull/12727/commits/07fd45ebe46bd2be735713a97fd233151c444e9c)